blocks/init/src/blocks/components/hamburger/assets/navigator.js

```diff
@@ -0,0 +1,6 @@
+export const getNavigatorVibrate = () => {
```

Comment:
put this here https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/helpers